packages/fastify-vite/config.js

```diff
@@ -80,19 +80,20 @@ const DefaultConfig = {
     return (error, req, reply) => {
       if (config.dev) {
         console.error(error)
+        scope.vite.devServer.ssrFixStacktrace(error);
       }
-      scope.vite.devServer.ssrFixStacktrace(error)
       scope.errorHandler(error, req, reply)
     }
   },
 }
 
 async function configure (options = {}) {
-  const defaultConfig = Object.create(DefaultConfig)
+  const defaultConfig = { ...DefaultConfig }
```

Comment:
it looks Object.create(DefaultConfig) was creating an empty object

Reply:
This is just to save memory, the new object appears empty but it's sharing a prototype.

```js
> const proto = {a:1}
undefined
> const obj = Object.create(proto)
undefined
> JSON.stringify(obj)
'{}'
> obj.a
1
```

Reply:
Ok cool feel free to discard it